v1.5/reference/etcd/data-model.md

```diff
@@ -1,600 +0,0 @@
----
```

Comment:
This is the v1.5 docs, not master.  Should we really be removing an entire doc from a previous release?